source/shared/net/guides/become-baker.rst

```diff
@@ -75,7 +75,7 @@ and send a ``baker add`` transaction to the network:
 
 .. code-block:: console
 
-   $concordium-client baker add MyBakerKeys.json --sender bakerAccount --stake <amount-to-stake> --open-delegation-for none --delegation-transaction-fee-commission 0.05 --delegation-baking-commission 0.05 --delegation-finalization-commission 1.0 --baker-url MyBakerURL.json --keys-in MyNewBakerKeys.json --keys-out <concordium-data-dir>/baker-credentials.json
+   $concordium-client baker add MyBakerKeys.json --sender bakerAccount --stake <amount-to-stake> --open-delegation-for all --delegation-transaction-fee-commission 0.1 --delegation-baking-commission 0.1 --delegation-finalization-commission 1.0 --baker-url MyBakerURL.json --keys-in MyNewBakerKeys.json --keys-out <concordium-data-dir>/baker-credentials.json
```

Comment:
I suggest using a proper URL for `--baker-url`, e.g. `https://example.com/baker`